src/static/helpers/slas.ts

```diff
@@ -92,27 +101,33 @@ export async function authorize(
     redirectURI: string;
     hint?: string;
     usid?: string;
+  } & CustomQueryParameters,
+  options?: {
+    headers?: { [key: string]: string };
```

Comment:
I think nodejs does have an official type `IncomingHttpHeaders`, but i haven't used it personally.